Formula/qt-gstreamer.rb

```diff
@@ -0,0 +1,37 @@
+class QtGstreamer < Formula
+  desc "QtGStreamer"
+  homepage "https://gstreamer.freedesktop.org/modules/qt-gstreamer.html"
+  url "https://gstreamer.freedesktop.org/src/qt-gstreamer/qt-gstreamer-1.2.0.tar.xz"
+  sha256 "9f3b492b74cad9be918e4c4db96df48dab9c012f2ae5667f438b64a4d92e8fd4"
+  head "https://anongit.freedesktop.org/git/gstreamer/qt-gstreamer.git"
+
+  depends_on "boost"
+  depends_on "glib"
+  depends_on "gstreamer"
+  depends_on "gst-plugins-base"
+  depends_on "pkg-config" => :build
+  depends_on "qt5"
+
+  def install
+    # QtGStreamer will NOT build on a case-insensitive HFS+ filesystem.
```

Comment:
https://bugzilla.gnome.org/show_bug.cgi?id=733788

It's proposed to use a separate directory for the offending headers, and then refer to them via full paths. Maybe that idea could be turned into working patch?